jenkins/bisect.jpl

```diff
@@ -390,16 +390,32 @@ def findMergeBase(kdir, good, bad) {
             script: "git merge-base --is-ancestor ${base} HEAD")
 
         if (good_base != 0) {
-            def ref = "${params.REF_KERNEL_TREE}/${params.REF_KERNEL_BRANCH}"
+
+            dir(kci_core) {
+                if (!(ref_kernel_url && ref_kernel_tree && params.ref_kernel_branch)) {
+
+                    ref = sh(script: """\
+                                    ./kci_build get_reference --tree-name ${params.KERNEL_TREE} \
+                                    --branch ${params.KERNEL_BRANCH}""", returnStdout: true).trim().tokenize("\n")
+
+                    if (ref.size() > 0) {
+                        ref_kernel_url = ref[0]
```

Comment:
Should be `ref_url`, `ref_tree` and `ref_branch` as that's what's used further down.

Reply:
Pushed missing corrections